### ISSUE TITLE
Already logged issues are not marked as logged

### ISSUE THREAD

**Issue by rbrekken** [simoneldevig/jira-toggl#43]
Hi,

I'm having a problem with already logged issues that are not marked as logged the next time I use the extension. Does anyone know the reason for that? This leads to duplicate worklogs if I do not check manually which issues are already logged.

**Comment by simoneldevig** [comment]
Hi @rbrekken 

Can you try and validate if the email you entered in the settings is the exact same that is used in JIRA because this is the one we're using for figuring out if the user has logged an entry or not :-)

**Comment by rbrekken** [comment]
Hi,

I've just validated my email address and it is exactly the same as in JIRA,
even when considering case sensitivity.

man. 7. mar. 2022 kl. 09:11 skrev Simon Eldevig ***@***.***>:

**Comment by simoneldevig** [comment]
@rbrekken alright. Are you using the merge option for entries with same description and project or do you post them separately? 

What happens when you log your time? Does the checkmark the appear in the extension?

**Comment by rbrekken** [comment]
No, I'm not using the merge option.("Merge time entries with same
comment"). No checkmark appears in the extension, and if I select already
logged entries, I get duplicate worklogs in Jira.

tor. 10. mar. 2022 kl. 09:29 skrev Simon Eldevig ***@***.***>:

**Comment by simoneldevig** [comment]
Thanks for clarification @rbrekken.

I've just tested it and there seems to be a bug when you're not using the merge option, then it can't find the correct worklog again.

I will try to find some time to look into the issue, but until then you can use the merge option to get your logged entries marked at logged, and then log everything in the end of the day.

**Comment by rbrekken** [comment]
Hi,

Finally, I got a chance to test this, and it is like you say; if I use the
merge option it works ok. I hope you can look into this since I would
prefer not to use the merge option.

tor. 10. mar. 2022 kl. 11:33 skrev Simon Eldevig ***@***.***>:

**Comment by simoneldevig** [comment]
I've just submitted

**Comment by simoneldevig** [comment]
@rbrekken I've just submitted a new version to Google which resolves this issue. Should be released sometime today :-)

**Comment by rbrekken** [comment]
Does this fix apply to the extension for Mozilla Firefox also?

tor. 24. mar. 2022 kl. 08:18 skrev Simon Eldevig ***@***.***>:

**Comment by simoneldevig** [comment]
@rbrekken yes
